djhtml/modes.py

```diff
@@ -65,7 +65,10 @@ def parse(self):
                 # token at the top of the stack.
                 if token.dedents:
                     try:
-                        if stack[-1].kind == token.kind and stack[-1].is_hard == token.is_hard:
+                        if (
+                            stack[-1].kind == token.kind
+                            and stack[-1].is_hard == token.is_hard
+                        ):
```

Comment:
These first two changes were automatically made by the `black` pre-commit hook.

Reply:
Confirmed, `pre-commit run --all` on `main` produces the same changes. Not sure why, Black's version hasn't changed (although we should upgrade Black because they have [released their first stable version](https://github.com/psf/black/releases/tag/22.1.0)!)